src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -26,18 +27,34 @@ const isPodCampaign = (cartItemCategory: string): boolean =>
   cartItemCategory.includes("tt-token");
 
 /**
- * Check if pod is chargeable
+ * Check if pod is chargeable.
+ *
+ * Redemption of the first token is chargeable for passport customer.
  *
+ * @param id customerId
  * @param identifiers policy identifiers
  * @param cartItem policy cart item
  */
 const isPodChargeable = (
+  id: string,
   identifiers: PolicyIdentifier[],
   cartItem: CartItem
-): boolean =>
-  identifiers.length > 0 &&
-  cartItem.category.includes("tt-token-lost") &&
-  cartItem.descriptionAlert === "*chargeable";
+): boolean => {
+  if (identifiers.length > 0) {
+    if (cartItem.category.includes("tt-token-lost")) {
+      if (cartItem.descriptionAlert === "*chargeable") {
+        return true;
+      }
+    } else if (validatePassport(id)) {
```

Comment:
Do we want to validate passport or accept the identificationFlag instead? Is this possible?

Reply:
do you mean the one from `useContext(IdentificationContext)`